SwerveControllerCommand/constants.py

```diff
@@ -0,0 +1,95 @@
+#
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
+
+from wpilib import TimedRobot
+from wpimath.geometry import Translation2d
+from wpimath.kinematics import SwerveDrive4Kinematics
+from wpimath.trajectory import TrapezoidProfile
+import math
+
+
+class DriveConstants:
+    kFrontLeftDriveMotorPort = 0
+    kRearLeftDriveMotorPort = 2
+    kFrontRightDriveMotorPort = 4
+    kRearRightDriveMotorPort = 6
+
+    kFrontLeftTurningMotorPort = 1
+    kRearLeftTurningMotorPort = 3
+    kFrontRightTurningMotorPort = 5
+    kRearRightTurningMotorPort = 7
+
+    kFrontLeftTurningEncoderPorts = [0, 1]
+    kRearLeftTurningEncoderPorts = [2, 3]
+    kFrontRightTurningEncoderPorts = [4, 5]
+    kRearRightTurningEncoderPorts = [6, 7]
+
+    kFrontLeftTurningEncoderReversed = False
+    kRearLeftTurningEncoderReversed = True
+    kFrontRightTurningEncoderReversed = False
+    kRearRightTurningEncoderReversed = True
+
+    kFrontLeftDriveEncoderPorts = [8, 9]
+    kRearLeftDriveEncoderPorts = [10, 11]
+    kFrontRightDriveEncoderPorts = [12, 13]
+    kRearRightDriveEncoderPorts = [14, 15]
+
+    kFrontLeftDriveEncoderReversed = False
+    kRearLeftDriveEncoderReversed = True
+    kFrontRightDriveEncoderReversed = False
+    kRearRightDriveEncoderReversed = True
+
+    kDrivePeriod = TimedRobot.kDefaultPeriod
```

Comment:
This file seems to be missing various important comments from the upstream example.